js/style/style_layer/extrusion_style_layer.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var util = require('../../util/util');
+var StyleLayer = require('../style_layer');
+
+function ExtrusionStyleLayer() {
+    StyleLayer.apply(this, arguments);
+}
+
+module.exports = ExtrusionStyleLayer;
+ExtrusionStyleLayer.prototype = util.inherit(StyleLayer, {});
```

Comment:
Alongside `CircleStyleLayer` for `circles`, `FillStyleLayer` for `fills`, etc, this class name would suggest the existence of an `extrusion` layer type. Could we change to `FillExtrusionStyleLayer`?